docs-mk/en/docs/build-nlp-model.md

```diff
@@ -26,7 +26,7 @@ Click on the *New Sentence* menu and enter the new sentence. The add a new user
  
 ### Declaring Entities
  
-If necessary, you can specify the entities of this new intent, by selecting the text of these entities, and then clicking on the "Add New Entity" button that has just appeared.
+If necessary, you can specify the entities of this new intent, by selecting the text of these entities, and then clicking on the "Choose or Add New Entity" button that has just appeared.
```

Comment:
I think that we should stick to one label. The new label is "Choose or Add **an** Entity" but in the documentation it is "Choose or Add **New** Entity"